Chaincase.UI/LiveSharp.dashboard.cs

```diff
@@ -0,0 +1,33 @@
+using LiveSharp;

```

Comment:
@DanGould Did we rm livesharp?
I think this got autogenerated

Reply:
yes, was this forked from master?